integration_test/bnbchain/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Start by using binance's node-binary dockerfile. This was built and uploaded to kava dockerhub.
+FROM kava/bnbchain:0.6.3
```

Comment:
can you rename `kava/bnbchain:0.6.3` to `deputy/bnbchain:0.6.3`, just for consistency. If we do not need to upload the docker image to dockerhub, can we delete the comment above?

Reply:
I can't rename "kava" to "deputy" because kava is the name of the docker hub account the image is pulled from.

What I can do is add the binance chain Dockerfile into this file, then we don't need to pull an image from kava's dockerhub account.

Reply:
alright, just leave it

Reply:
I replaced the dockerfile with a way simpler one. It removes the kava dependency and reduces build times.